model/flow/execution_receipt.go

```diff
@@ -28,6 +30,7 @@ func ExecutionReceiptFromMeta(meta ExecutionReceiptMeta, result ExecutionResult)
 type ExecutionReceipt struct {
 	ExecutorID        Identifier
 	ExecutionResult   ExecutionResult
+	ResultSignature   crypto.Signature // signature only over the ExecutionResultID; for aggregation in Seal
```

Comment:
Does this mean the seal will contain signatures of all executors over the execution result ID?

Reply:
Leo asked the same question. Here the [answer](https://github.com/onflow/flow-go/pull/319#discussion_r564029277): 
> In the original protocol spec, we also planned to aggregate EN's signatures to reward them for their work. The idea was to reward all ENs that had produced a compatible result by the time the result is sealed (and include their signatures in the seal).
> 
> As we are now incorporating the Receipts directly in the chain, this is not necessary anymore as for calculating rewards, we can just inspect the chain payload.

Reply:
removed `ExecutionReceipt.ResultSignature` entirely